### PR TITLE
ocrmypdf: move to python3Packages

### PR DESCRIPTION
**Pull request by dotlambda**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

cc @antifuchs @alarsyo

### CODE REVIEW

File: pkgs/applications/office/paperless-ng/default.nix [hunk 1]
```diff
@@ -5,7 +5,6 @@
 , ghostscript
 , imagemagick
 , jbig2enc
-, ocrmypdf
```
Comment (antifuchs) [inline pkgs/applications/office/paperless-ng/default.nix:8]: Does paperless-ng still work without ocrmypdf? I haven't used it in a while (and it seems to fail to build anyway), but seems worth checking with users if there are any left.

**Review by antifuchs — APPROVED** [review]
Yep, this seems right to me: it is a python package after all.

File: pkgs/applications/office/paperless-ng/default.nix (hunk 1 above)
Comment (alarsyo) [reply pkgs/applications/office/paperless-ng/default.nix:8]: As far as I know this is still needed, yes (see their [requirements](https://github.com/jonaswinkler/paperless-ng/blob/7bc8325df910ab57ed07849a3ce49a3011ba55b6/requirements.txt#L58)).

Comment (alarsyo) [reply pkgs/applications/office/paperless-ng/default.nix:8]: also regarding

> seems worth checking with users if there are any left

why wouldn't there be any users left? Am I missing something? :sweat_smile:

Comment (antifuchs) [reply pkgs/applications/office/paperless-ng/default.nix:8]: agh, sorry, this came out more snide than I intended: I'd been using paperless-ng a while ago, and got a bit frustrated with how often its build broke, so I switched away to docspell. It's very possible there are people using it!

Comment (alarsyo) [reply pkgs/applications/office/paperless-ng/default.nix:8]: I've just built it successfully from nixpkgs's master branch, no problems here :/

Comment (dotlambda) [reply pkgs/applications/office/paperless-ng/default.nix:8]: Paperless-ng will still use `ocrmypdf`, just from `python3Packages`.

**Pull request merged**